### DISCUSSION TITLE
Performance vs explorer.exe

### DISCUSSION THREAD

**Discussion by VictorieeMan** [eythaann/Seelen-UI#74] [General]
Hi there! I just learned about this custom UI. Tell me, how does it perform in comparison to the default UI in explorer.exe?

Any benchmarks done?

**Comment by eythaann** [comment]
I haven’t done a benchmark on performance yet, and the goal is not to be faster or more lightweight than explorer.exe. On the contrary, the objective is to add as many layers of customization as possible, so Seelen prioritizes design over performance. Of course, we try to make it as resource-friendly as possible, but that’s not the focus of this project.
